.github/workflows/push.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     secrets: inherit
 
   build:
-    uses: ./.github/workflows/client.yml
+    uses: ./.github/workflows/core.yml
     needs: [clippy_check, cargo_test]
```

Comment:
Same here

Reply:
Don't think so - it's using the job name, referred to earlier as `clippy_check`
```
jobs:
  clippy_check:
    uses: ./.github/workflows/checks.yml
    secrets: inherit
```